scripts/rust-tests.sh

```diff
@@ -1,10 +1,18 @@
 #!/usr/bin/env bash
 # Runs the all the Rust integration tests
 
-set -euxo pipefail
+set -euo pipefail
 export RUST_LOG="${RUST_LOG:-info}"
 
-source ./scripts/setup-tests.sh ""
+# Convert RUST_LOG to lowercase
+# if RUST_LOG is none, don't show output of test setup
+if [ "${RUST_LOG,,}" = "none" ]; then
```

Comment:
Why convert it to lowercase? Why would would anyone ... `RUST_LOG=nOnE`?

Reply:
I've seen people do all uppercase, so `RUST_LOG=NONE`.

Reply:
I agree :D especially because you have to hold the shift key while typing (nobody is using capslock hopefully :sweat_smile: )